changelog/fragments/liveness_readiness_probe_for_operator.yaml

```diff
@@ -0,0 +1,91 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Helm-based operators, added Liveness and Readiness probe by default using [`healthz.Ping`](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/healthz#CheckHandler).
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: (Optional) For Helm-based operators, add Liveness and Readiness probe
+      body: >
+        New projects built with the tool will have the probes configured by default. The endpoints `/healthz` and
+        `/readyz` are available now in the image based provided.
+
+        You can update your pre-existing project to use them. For that update the Dockerfile to use the latest
+        release base image, then add the following to the `manager` container in
+        `config/default/manager/manager.yaml`:
```

Comment:
The path to the manager resource seems to be wrong. Should be `config/manager/manager.yaml` instead.